bazel/repositories.bzl

```diff
@@ -74,10 +74,10 @@ def proxy_wasm_cpp_host_repositories():
     maybe(
         http_archive,
         name = "rules_rust",
-        sha256 = "f3d443e9ad1eca99fbcade1c649adbd8200753cf22e47846b3105a43a550273b",
-        strip_prefix = "rules_rust-0.8.1",
+        sha256 = "dc8d79fe9a5beb79d93e482eb807266a0e066e97a7b8c48d43ecf91f32a3a8f3",
```

Comment:
This doesn't match the SHA256 of the downloaded file on my machine, which is: `c8a84a01bff4be4c7a8beefbc12e713ff296fed2110c30572a44205856594cfc`.

This is why CI fails.